rdt-open-reader/src/test/java/io/ona/rdt/robolectric/util/UtilsTest.java

```diff
@@ -223,4 +225,26 @@ public void testIsValidJSONObjectShouldReturnCorrectStatus() {
         Assert.assertFalse(Utils.isValidJSONObject(null));
         Assert.assertFalse(Utils.isValidJSONObject(""));
     }
+
+    @Test
+    public void testVerifyUserAuthorizationShouldExecuteAuthorizationTask() {
+
+        // verify pending state
+        Utils.UserAuthorizationVerificationTask userAuthorizationVerificationTask = Mockito.mock(Utils.UserAuthorizationVerificationTask.class);
+        ReflectionHelpers.setStaticField(Utils.UserAuthorizationVerificationTask.class, "INSTANCE", userAuthorizationVerificationTask);
+        Mockito.when(userAuthorizationVerificationTask.getStatus()).thenReturn(AsyncTask.Status.PENDING);
+        Utils.verifyUserAuthorization(RDTApplication.getInstance());
+        Mockito.verify(userAuthorizationVerificationTask, Mockito.times(1)).execute();
```

Comment:
This verification is already taken care of https://github.com/onaio/rdt-standard/pull/945/files#diff-92bb3b1d837c9f367cd89c386e8a01e1902d58d39b106f850196353f98fb9a19R243

Reply:
fixed.